core/src/main/java/org/pentaho/reporting/platform/plugin/ParameterDependencyGraph.java

```diff
@@ -140,6 +139,18 @@ public void setAllParameterNames( Set<String> allParameterNames ) {
     return Collections.unmodifiableSet( dependencyGraph.keySet() );
   }
 
+  /**
+   * Stream and verify that the parameter exists as a dependency in the dependency graph
```

Comment:
"Stream and verify" - streaming is about the particular implementation, no? Don't think it needs to surface in the description.

Reply:
Will remove stream, leave verify